intakebuilder/gfdlcrawler.py

```diff
@@ -31,19 +31,12 @@ def crawlLocal(projectdir, dictFilter,dictFilterIgnore,logger):
                dictInfo = {}
                dictInfo = getinfo.getProject(projectdir, dictInfo)
```

Comment:
This works today because "dev" is the activity_id for any PP data that's in /archive or has "pp" in it. This needs to be tested later on ,under a different case.